Errors after upgrade to 1.0

I updated auto-iimport from 0.2.6 to 1.0.0 and i got the following error:

`ERROR in ./tmp/ember_auto_import_webpack-staging_dir-r6pCK2QC.tmp/entry.js
Module not found: Error: Can't resolve 'C:UsersadminDocumentsOculariumOculariumFrontend 2.0OculariumFrontendCore
ode_modulesocularium-sharedlibscenarioManager.js' `

Looks like the slashes are gone, the correct path should be like this:

`
C:Users/admin/Documents/Ocularium/OculariumFrontend 2.0/OculariumFrontendCore
ode_modules/ocularium-shared/lib/scenarioManager.js' 
`

> Hash: 56af320e769258f9084b
Version: webpack 4.12.1
Time: 72ms
Built at: 2018-06-25 09:57:04
 Asset      Size  Chunks             Chunk Names
app.js  4.52 KiB    main  [emitted]  main
Entrypoint main = app.js

Log file
```
=================================================================================

ENV Summary:

  TIME: Tue Jun 26 2018 11:51:16 GMT+0200 (Romance Daylight Time)
  TITLE: ember
  ARGV:
  - C:\Program Files\nodejs\node.exe
  - C:\Users\admin\AppData\Roaming\npm\node_modules\ember-cli\bin\ember
  - s
  EXEC_PATH: C:\Program Files\nodejs\node.exe
  TMPDIR: C:\Users\admin\AppData\Local\Temp
  SHELL: null
  PATH:
  - C
  - \Program Files\Docker\Docker\Resources\bin;C
  - \Windows\system32;C
  - \Windows;C
  - \Windows\System32\Wbem;C
  - \Windows\System32\WindowsPowerShell\v1.0\;C
  - \Program Files\Git\cmd;C
  - \Program Files\PuTTY\;C
  - \ProgramData\ComposerSetup\bin;C
  - \Program Files (x86)\Skype\Phone\;C
  - \Users\admin\AppData\Roaming\nvm;C
  - \Program Files\nodejs;C
  - \Program Files\nodejs\;C
  - \Users\admin\AppData\Local\Microsoft\WindowsApps;C
  - \PHP;C
  - \Users\admin\AppData\Roaming\Composer\vendor\bin;C
  - \Program Files\Microsoft VS Code\bin;C
  - \Users\admin\AppData\Roaming\nvm;C
  - \Users\admin\AppData\Roaming\npm
  PLATFORM: win32 x64
  FREEMEM: 8131403776
  TOTALMEM: 17112006656
  UPTIME: 11703.1337309
  LOADAVG: 0,0,0
  CPUS:
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  - Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz - 3392
  ENDIANNESS: LE
  VERSIONS:
  - ares: 1.10.1-DEV
  - cldr: 32.0
  - http_parser: 2.8.0
  - icu: 60.1
  - modules: 57
  - napi: 3
  - nghttp2: 1.32.0
  - node: 8.11.3
  - openssl: 1.0.2o
  - tz: 2017c
  - unicode: 10.0
  - uv: 1.19.1
  - v8: 6.2.414.54
  - zlib: 1.2.11

ERROR Summary:

  - broccoliBuilderErrorStack: Error: webpack returned errors to ember-auto-import
    at webpack.run (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\ember-auto-import\js\webpack.js:67:32)
    at finalCallback (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:157:39)
    at hooks.done.callAsync.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:206:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\tapable\lib\Hook.js:35:21)
    at emitRecords.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:204:22)
    at Compiler.emitRecords (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:319:39)
    at emitAssets.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:198:10)
    at hooks.afterEmit.callAsync.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:305:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:6:1)
  - codeFrame: webpack returned errors to ember-auto-import
  - errorMessage: Build Canceled: Broccoli Builder ran into an error with `BundlerPlugin` plugin. 💥
webpack returned errors to ember-auto-import
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
    - treeDir: [undefined]
  - message: Build Canceled: Broccoli Builder ran into an error with `BundlerPlugin` plugin. 💥
webpack returned errors to ember-auto-import
  - name: Error
  - nodeAnnotation: BundlerPlugin
  - nodeName: BundlerPlugin
  - originalErrorMessage: webpack returned errors to ember-auto-import
  - stack: Error: webpack returned errors to ember-auto-import
    at webpack.run (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\ember-auto-import\js\webpack.js:67:32)
    at finalCallback (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:157:39)
    at hooks.done.callAsync.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:206:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook [as _callAsync] (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\tapable\lib\Hook.js:35:21)
    at emitRecords.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:204:22)
    at Compiler.emitRecords (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:319:39)
    at emitAssets.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:198:10)
    at hooks.afterEmit.callAsync.err (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\webpack\lib\Compiler.js:305:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\admin\Documents\Ocularium\OculariumFrontend 2.0\OculariumFrontendCore\node_modules\tapable\lib\HookCodeFactory.js:24:12), <anonymous>:6:1)

=================================================================================
```

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
     "enhanced-resolve": "^4.0.0",
     "fs-tree-diff": "^0.5.7",
     "handlebars": "^4.0.11",
+    "js-string-escape": "^1.0.1",
     "lodash": "^4.17.10",
     "mkdirp": "^0.5.1",
     "quick-temp": "^0.1.8",

--- a/ts/webpack.ts
+++ b/ts/webpack.ts
@@ -3,12 +3,15 @@ import { join, dirname, basename } from 'path';
 import { merge } from 'lodash';
 import quickTemp from 'quick-temp';
 import { writeFileSync } from 'fs';
-import { compile } from 'handlebars';
+import { compile, registerHelper } from 'handlebars';
+import jsStringEscape from 'js-string-escape';
+
+registerHelper('js-string-escape', jsStringEscape);
 
 const entryTemplate = compile(`
 module.exports = (function(){
   {{#each modules as |module|}}
-    window.define('{{module.specifier}}', [], function() { return require('{{module.entrypoint}}'); });
+    window.define('{{js-string-escape module.specifier}}', [], function() { return require('{{js-string-escape module.entrypoint}}'); });
   {{/each}}
 })();
 `);
@@ -41,12 +44,8 @@ export default class WebpackBundler {
   }
 
   private writeEntryFile(modules){
-    let moduleList = Object.keys(modules).map(specifier => ({ specifier, entrypoint: this.normalizePath(modules[specifier].entrypoint) }));
+    let moduleList = Object.keys(modules).map(specifier => ({ specifier, entrypoint: modules[specifier].entrypoint }));
     writeFileSync(join(this.stagingDir, 'entry.js'), entryTemplate({ modules: moduleList }));
-  }
-
-  private normalizePath(entrypoint) {
-    return entrypoint.replace(/\\/g, '\\\\');
   }
 
   private async runWebpack(){

--- a/ts/webpack.ts
+++ b/ts/webpack.ts
@@ -41,8 +41,12 @@ export default class WebpackBundler {
   }
 
   private writeEntryFile(modules){
-    let moduleList = Object.keys(modules).map(specifier => ({ specifier, entrypoint: modules[specifier].entrypoint}));
+    let moduleList = Object.keys(modules).map(specifier => ({ specifier, entrypoint: this.normalizePath(modules[specifier].entrypoint) }));
     writeFileSync(join(this.stagingDir, 'entry.js'), entryTemplate({ modules: moduleList }));
+  }
+
+  private normalizePath(entrypoint) {
+    return entrypoint.replace(/\\/g, '\\\\');
   }
 
   private async runWebpack(){

